src/main/java/org/aksw/iguana/commons/rdf/IONT.java

```diff
@@ -20,11 +19,7 @@ public class IONT {
     public static final Resource metric = ResourceFactory.createResource(NS + "Metric");
 
     public static Resource getMetricClass(Metric metric) {
-        // TODO: compare with stresstest class
+        // TODO: compare with stresstest class (stresstest class as a subclass of Task is iont:Stresstest while QPS for example is iont:metric/QPS)
```

Comment:
I don't get this comment. Hope it's not originally from me :)

Reply:
The inherited metric classes like QPS have the uri `http://iguana-benchmark.eu/class/metric/QPS`, while the stresstest, which inherits from the class task, only has the uri `http://iguana-benchmark.eu/class/Stresstest`. The difference is, that the metric class has a `metric/`-"prefix". That was what I wanted to point out. Maybe we should give the stresstest a `task/`-prefix as well?

Reply:
What do we actually use the metric classes for? Shouldn't the properties be enough?

Reply:
The metric classes give the metric properties their type. 
![grafik](https://github.com/dice-group/IGUANA/assets/32801560/a57126b1-03c0-4aa4-827d-5bd2e1508c1c)